### PR TITLE
Bugsnag device mode updates for iOS SDK v2

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Bugsnag device mode updates for iOS SDK v2

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by pallabmaiti — APPROVED** [review]

File: docs/destinations/error-reporting/bugsnag.mdx [hunk 1]
```diff
@@ -172,44 +126,100 @@ RudderConfigBuilder *builder = [[RudderConfigBuilder alloc] init];
 ```
 </span>
 </li>
-</ul>
+</ol>
+      </TabPanel>
+       <TabPanel>
+          <div class="warningBlock">
+          This device mode integration is supported for Bugsnag v8.15.0 and above.
```
Comment (pallabmaiti) [inline docs/destinations/error-reporting/bugsnag.mdx:133]:
```suggestion
          This device mode integration is supported for Bugsnag v6.16.4 and above.
```

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

**Comment by pallabmaiti** [comment]
Words used in steps for Bugsnag are very much different from [Firebase](https://www.rudderstack.com/docs/destinations/analytics/firebase/#adding-device-mode-integration). Did we intentionally do that?

**Review by pallabmaiti — APPROVED** [review]

**Comment by ameypv-rudder** [comment]
> Words used in steps for Bugsnag are very much different from [Firebase](https://www.rudderstack.com/docs/destinations/analytics/firebase/#adding-device-mode-integration). Did we intentionally do that?

@pallabmaiti please confirm if you are referring to the iOS v2 tab? Because the steps appear to be quite similar to Firebase.

**Comment by pallabmaiti** [comment]
> @pallabmaiti please confirm if you are referring to the iOS v2 tab? Because the steps appear to be quite similar to Firebase.
I had a call with @ishachopra0401 to explain. It is sorted now.

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**